Tests/FluentAssertions.Specs/Specialized/TaskAssertionSpecs.cs

```diff
@@ -104,6 +104,26 @@ public async Task When_task_completes_late_it_should_fail()
             // Assert
             await action.Should().ThrowAsync<XunitException>();
         }
+
+        [Fact]
+        public async Task Cancelling_a_token_also_means_the_task_is_completed()
```

Comment:
Should we add a similar test for the other two methods that now pass along `Task.CompletedTask`?
Especially `NotCompleteWithinAsync` as negated logic always triggers me.

Reply:
They were already covered by tests, aren't they?